src/script/components/calling/ParticipantMicOnIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {css, keyframes} from '@emotion/core';
+import {registerReactComponent} from 'Util/ComponentUtil';
+import SVGProvider from '../../auth/util/SVGProvider';
+
+const fadeAnimation = keyframes`
+  0%   { opacity: 0; }
+  100% { opacity: 1; }
+`;
+
+interface ParticipantMicOnIconProps {
+  activeColor?: string;
+  className?: string;
+  isActive?: boolean;
+}
+
+const ParticipantMicOnIcon: React.FC<ParticipantMicOnIconProps> = ({
+  className,
+  isActive = false,
+  activeColor = '#fff',
+  ...props
+}) => {
+  return (
+    <span
+      css={css`
```

Comment:
No need to call extra `css` here. `css={{...}}` would be enough.

Reply:
nice catch, https://github.com/wireapp/wire-webapp/pull/10094/commits/a4d957a9133cd531722a0027e338dc1060c57599